datanode/networkhistory/service_test.go

```diff
@@ -1264,17 +1264,23 @@ func getDatabaseDataSummary(ctx context.Context, connConfig sqlstore.ConnectionC
 	}
 
 	for table, meta := range dbMetaData.TableNameToMetaData {
+		if len(meta.SortOrder) == 0 {
+			meta.SortOrder = "vega_time, seq_num"
+		}
 		summary := tableDataSummary{tableName: table}
 		err = conn.QueryRow(ctx, fmt.Sprintf("select count(*) from %s", table)).Scan(&summary.rowCount)
 		if err != nil {
 			panic(err)
 		}
 
 		if summary.rowCount > 0 {
-			err = conn.QueryRow(ctx, fmt.Sprintf("SELECT md5(CAST((array_agg(f.* order by %s))AS text)) FROM %s f; ",
-				meta.SortOrder, table)).Scan(&summary.dataHash)
+			q := fmt.Sprintf("SELECT md5(CAST((array_agg(f.* order by %s))AS text)) FROM %s f; ", meta.SortOrder, table)
+			err = conn.QueryRow(ctx, q).Scan(&summary.dataHash)
+			// err = conn.QueryRow(ctx, fmt.Sprintf("SELECT md5(CAST((array_agg(f.* order by %s))AS text)) FROM %s f; ",
```

Comment:
looks like the same thing as original?  could we restore the commented out code?